worlds/sm64ex/Options.py

```diff
@@ -127,6 +127,30 @@ class MoveRandomizerActions(OptionSet):
     valid_keys = [action for action in action_item_table if action != 'Double Jump']
     default = valid_keys
 
+sm64_options_groups = [
+    OptionGroup("Sanity Options", [
```

Comment:
Maybe call this logic options or something like that? "Sanity" doesn't really describe the contents super well

Reply:
Item Options?

Reply:
Probably not that, since the last group is always "Item & Location Options"